omp/test/matrix/dense_kernels.cpp

```diff
@@ -51,6 +50,7 @@ OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
 
 #include "core/matrix/dense_kernels.hpp"
```

Comment:
maybe this one should go top instead of the `dense.hpp` again.

Reply:
That is true, this should go on top.